src/blueprint/operators/grpc.rs

```diff
@@ -249,7 +256,7 @@ mod tests {
         assert!(result.is_err());
         assert_eq!(
             result.err().unwrap(),
-            ValidationError::new("Invalid method format: package_name.ServiceName. Expected format is <package/proto_id>.<service>.<method>".to_string())
+            ValidationError::new("Invalid method format: package_name.ServiceName. Expected format is <package>.<service>.<method>".to_string())
```

Comment:
```suggestion
            ValidationError::new("Invalid method format: package_name.ServiceName. Expected format is <package>.<service>/<method>".to_string())
```

Reply:
marking it as resolved as it was resolved on discord.